dynamic.js

```diff
@@ -47,4 +47,133 @@ function closecContainer() {
 icon.addEventListener('click', openContainer);
 closeBtn.addEventListener('click', closecContainer);
 const navLink = document.querySelectorAll('.nav-click');
-navLink.forEach((a) => a.addEventListener('click', closecContainer));
\ No newline at end of file
+navLink.forEach((a) => a.addEventListener('click', closecContainer));
+
+// pop up window info
+
+const mockData = [
+  {
+    headerCard: 'Profesional Art Printing Data More',
+    textCard: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    badgesCard: ['html', 'bootstrap', 'Ruby'],
+    imageURL: './images/popup-img.png',
+    liveURL: 'https://www.github.com',
+    projectURL: 'https://www.github.com/ahmedadel56',
+  },
```

Comment:
- [ ] [OPTIONAL] As mentioned in the project requirements :
![image](https://user-images.githubusercontent.com/43698511/132774628-69e8571d-2b67-4f5a-bd4e-125f03f41536.png)
You need to remove all the HTML code for the portfolio section and generate it using JavaScript. I mentioned this as optional only because the first reviewer missed this point however I really recommend you make this change. Adding a new project in the future will be easy, Instead of creating a new HTML block with the project's information, you'll only have to update the mockData object by adding the new project's information.